util/topgen/intermodule.py

```diff
@@ -530,6 +530,9 @@ def im_netname(obj: OrderedDict, suffix: str = "") -> str:
         if obj["act"] == "rsp" and suffix == "req":
             return "{package}::{struct}_REQ_DEFAULT".format(
                 package=obj["package"], struct=obj["struct"].upper())
+        if obj["act"] == "rcv" and suffix == "":
```

Comment:
@eunchan 

let me know if you're okay with this change. I had to modify this because the `uni` `rcv` types were getting a default value without this change.

Reply:
... Thanks for catching it!